lib/gui/app.js

```diff
@@ -264,11 +262,10 @@ app.config(($urlRouterProvider) => {
 });
 
 app.config(($provide) => {
-  $provide.decorator('$exceptionHandler', ($delegate, $injector) => {
-    return (exception, cause) => {
-      const ErrorService = $injector.get('ErrorService');
-      ErrorService.reportException(exception);
-      $delegate(exception, cause);
+  $provide.decorator('$exceptionHandler', ($delegate) => {
+    return (except, cause) => {
```

Comment:
Hm, I wonder if this means we should pick another name for the `exception` module. What about `exceptionReporter`?